staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources.go

```diff
@@ -171,6 +185,7 @@ func (o *APIResourceOptions) RunAPIResources() error {
 	}
 
 	resources := []groupResource{}
+	var filteredList []*metav1.APIResourceList
```

Comment:
Can you rename the `resources` to `allResources`, so it's clear which one is what.